vpr/src/base/vpr_types.h

```diff
@@ -164,6 +164,7 @@ enum e_block_pack_status {
     BLK_PASSED,
     BLK_FAILED_FEASIBLE,
     BLK_FAILED_ROUTE,
+    BLK_FAILED_FLOORPLANNING,
```

Comment:
think about whether this enum can be in a lower header file. If it's only used in the clusterer, can have a cluster_utilities header file and put this enum in there. This can be a cleanup on a separate pull request

Reply:
in pack subdirectory - is there a cluster utilities type header file? check, could possibly put there